src/orders/model/RefundTransaction.ts

```diff
@@ -0,0 +1,8 @@
+import TransactionStatus from './TransactionStatus';
+
+export default interface RefundTransaction {
```

Comment:
Just call this Transaction.ts and rename the model to Transaction. No need to specify that it's for a refund now. If we do add support for these on other parts of the order we should be able to reuse the same object